### PR TITLE
Updates for Environment Canada camera component

### PR DESCRIPTION
**Pull request by michaeldavie**
## Proposed change

Documents changes to `environment_canada` configuration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/34729
- This PR fixes or closes issue: https://github.com/michaeldavie/env_canada/issues/17

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by michaeldavie** [comment]
Sorry, I don't understand the build error.

File: source/_integrations/environment_canada.markdown [hunk 1]
```diff
@@ -208,8 +210,8 @@ loop:
   default: true
   type: boolean
 precip_type:
-  description: Determines whether to use the intensity bands for rain or snow. Valid values are RAIN and SNOW.
+  description: Determines whether to use the intensity bands for rain or snow. Valid values are `rain` and `snow`.
   required: false
   type: string
-  default: RAIN from April to October, SNOW from November to March
+  default: `rain` from April to October, `snow` from November to March
```
Comment (klaasnicolaas) [inline source/_integrations/environment_canada.markdown]:
```suggestion
  default: "`rain` from April to October, `snow` from November to March"
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
This should fix the build error

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing PR, as the parent PR has been closed.

**Pull request closed without merging**